config/sidekiq.yml

```diff
@@ -3,18 +3,17 @@
 :concurrency: 32
 :logfile: ./log/sidekiq.json.log
 :timeout: 4
-
 # Any changes to the queues should be reflected in `docs/queues.md`.
-:queues:
-  - delivery_transactional
-  - [delivery_immediate_high, 10]
-  - [delivery_immediate, 5]
+# See https://github.com/mperham/sidekiq/wiki/Advanced-Options#queues
```

Comment:
I think we've lost the `queues:` key somewhere in this diff...

Reply:
Oops. It's back. This is "one of those" PRs, isn't it.